### PR TITLE
v1.16.0

### PR DESCRIPTION
**Pull request by ashie**
**Which issue(s) this PR fixes**: 
Fixes #

**What this PR does / why we need it**: 

**Docs Changes**:

**Release Note**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
+# v1.16
+
+## Release v1.16.0 - 2023/03/14
+
+### Enhancement
+
+* Add method for testing Pfiltered_with_time`
```
Comment (cosmo0920) [inline CHANGELOG.md:7]: `P` should be ` ?

Comment (ashie) [reply CHANGELOG.md:7]: Oops, thanks for pointing out it!

Comment (ashie) [reply CHANGELOG.md]: Fixed.

**Comment by ashie** [comment]
I postpone the release date to March 29th, it should be the final answer...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+
+### Bug Fix
+
+* http_server_helper: Fix format of log messages originating from Async gem
+  https://github.com/fluent/fluentd/pull/3987
+* Change to not generate a sigdump file after receiving a SIGTERM signal on
+  non-Windows
+  https://github.com/fluent/fluentd/pull/4034
+  https://github.com/fluent/fluentd/pull/4043
+* out_forward: fix error of ack handling confict on stopping with
```
Comment (daipom) [inline CHANGELOG.md:24]:
```suggestion
* out_forward: fix error of ack handling conflict on stopping with
```

Sorry, this is my typo.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+  https://github.com/fluent/fluentd/pull/4030
+* Fix problem that some system configs are not reflected
+  https://github.com/fluent/fluentd/pull/4064
+  https://github.com/fluent/fluentd/pull/4065
+  https://github.com/fluent/fluentd/pull/4086
+  https://github.com/fluent/fluentd/pull/4090
+  https://github.com/fluent/fluentd/pull/4096
+* Fix bug that the logger outputs some initial log messages with some settings not applied, such as format.
+  https://github.com/fluent/fluentd/pull/4091
+* Windows: Fix a bug that the wrong log file is reopened with log ratote setting
```
Comment (daipom) [inline CHANGELOG.md:35]:
```suggestion
* Windows: Fix a bug that the wrong log file is reopened with log rotate setting
```

Sorry, this is too.

**Comment by daipom** [comment]
Do we need to include 66c4a8b40310d8fe808e46ac4ce6b3af13157cda?
This commit seems to be not included in any PR.

**Comment by ashie** [comment]
Thanks for notifying it, I've added it with the PR number #3961

File: CHANGELOG.md (hunk 2 above)
Comment (ashie) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 3 above)
Comment (ashie) [reply CHANGELOG.md]: Fixed.

**Review by daipom — APPROVED** [review]
LGTM. Thanks for summarizing this!

**Pull request merged**